tests/sentry/models/test_lastappconnectbuildscheck.py

```diff
@@ -0,0 +1,156 @@
+import uuid
+from datetime import datetime
+
+import pytest
+from django.utils import timezone
+
+from sentry.lang.native import appconnect
+from sentry.models import LatestAppConnectBuildsCheck
+
+
+class TestCreateOrUpdateBuildCheck:
+    @pytest.fixture
+    def config(self):
+        return appconnect.AppStoreConnectConfig(
+            type="appStoreConnect",
+            id=uuid.uuid4().hex,
+            name="Apple Honk Store Connect",
+            appconnectIssuer="abc123" * 6,
+            appconnectKey="abc123key",
+            appconnectPrivateKey="----BEGIN PRIVATE KEY---- honkhonk",
+            itunesUser="me@example.com",
+            itunesPassword="secrethonk",
+            itunesSession="THE-COOKIE",
+            itunesCreated=datetime.utcnow(),
+            appName="honk beep",
+            appId="123",
+            bundleId="com.honk.app",
+            orgPublicId="71105f98-7743-4844-ab70-2c901e2ea13d",
+            orgName="Honk Com",
+        )
+
+    @pytest.mark.django_db
+    def test_add_new_check_date(self, default_project, config):
```

Comment:
I’m all 👍🏻 on tests, however all these just assert that the database model is, well, behaving like a database. Can we actually test the business logic here somehow?
I wish we could easily write an integration test for this, but that would require us to mock all of the API calls to apple currently so is not really feasible either :-(

Reply:
I've reworked the task a little so I could write tests for the build processing logic (creating entries in DB, writing check date to DB, filtering builds that we have the dsyms for) - let me know what you think of those!